sycl/test/matrix/matrix-int8-uint8-test.cpp

```diff
@@ -0,0 +1,168 @@
+// RUN: %clangxx -march=sapphirerapids -fsycl -O2 %s -o %t.out
```

Comment:
TBH I don't see much of a value of these tests in this repository. Please consider moving them to https://github.com/intel/llvm-test-suite

Reply:
`-march=sapphirerapids`
What about other targets?

Reply:
-march=sapphirerapids is not needed for JIT tests, right Bing?

Reply:
BTW, I added these tests to https://github.com/intel/llvm-test-suite/pull/443/
Bing, let's just keep the old ones. We don't need to add new ones.

Reply:
@dkhaldi, has your comment above been addressed?
Otherwise, looks good to me.

Reply:
@v-klochkov, yes the comment above has been addressed:
- The new tests were removed from here and moved to llvm-test-suite
- -march=sapphirerapids has been removed from the JIT tests